resources/js/components/sponsors/SponsorLogin.vue

```diff
@@ -0,0 +1,160 @@
+        <img 
+            src="/img/ic_robobuzz_white_outline.svg"
+            alt="RoboJackets Logo"
+            height="200"
+            />
+    </div>
+  <div class="card shadow-sm p-4">
+    <h3 class="text-center mb-4">Sponsor Login</h3>
+
+    <form @submit.prevent="handleSubmit">
+      <transition name="slide-fade" mode="out-in">
+        <div v-if="!emailValidated" key="email-step">
+          <label for="email" class="form-label">Email</label>
+          <input 
+            type="email" 
+            v-model="email" 
+            class="form-control" 
+            placeholder="you@company.com" 
+            required 
+            @keydown.enter.prevent="validateEmail">
+          <button type="button" class="btn btn-primary mt-3" @click="validateEmail">Next</button>
+        </div>
+
+        <div v-if="emailValidated" key="password-step">
+          <label for="password" class="form-label">One-Time Password</label>
+          <input 
+            type="password" 
+            v-model="password" 
+            class="form-control" 
+            placeholder="Enter the one-time password emailed to you." 
+            required>
+          <button type="submit" class="btn btn-success mt-3">Login</button>
+        </div>
+      </transition>
+    </form>
+  </div>
+  <transition name="slide-fade">
+      <div v-if="emailValidated" class="resend-section">
+        <button
+          class="btn btn-link"
+          :disabled="!canResend"
+          @click="sendOTP"
+        >
+          {{ canResend ? 'Resend OTP' : `Resend in ${resendCooldown}s` }}
+        </button>
+      </div>
+    </transition>
+  </div>
+</template>
+
+<script>
+import Swal from 'sweetalert2';
+import axios from 'axios';
+
+export default {
+  name: 'SponsorLogin',
+  data() {
+    return {
+      email: '',
+      password: '',
+      emailValidated: false,
+      canResend: false,
+      resendTimer: null,
+      resendCooldown: 30,
+    };
+  },
+  methods: {
+    showToast(message, icon = 'info') {
+      Swal.fire({
+        toast: true,
+        position: 'bottom-end',
+        icon,
+        title: message,
+        showConfirmButton: false,
+        timer: 3000,
+        timerProgressBar: true
+      });
+    },
+    validateEmail() {
+    //   axios.post('/sponsor/check-email', { email: this.email })
+    //     .then(res => {
+    //       if (res.data.valid) {
+    //         this.emailValidated = true;
+    //         this.showToast('A one-time password has been sent to your email.', 'info');
+    //       } else {
+    //         this.showToast('Email domain not approved.', 'error');
+    //       }
+    //     })
+    //     .catch(() => {
+    //       this.showToast('Something went wrong. Please try again.', 'error');
+    //     });
+        if (this.email === 'gpburdell3@gatech.edu') {
+            this.emailValidated = true;
+            this.sendOTP();
+        } else {
+            this.showToast('Email domain not approved.', 'error');
```

Comment:
We can fix later but this may be better as an alert modal vs a toast that will disappear, but I'm also concerned that this error message doesn't tell people who to contact to resolve the issue